[clang-tidy] Ignore `if consteval` in else-after-return (#91588)
llvm/llvm-project: clang-tools-extra/docs/ReleaseNotes.rst

  <clang-tidy/checks/readability/duplicate-include>` check by excluding include
  directives that form the filename using macro.

- Improved :doc:`readability-else-after-return
  <clang-tidy/checks/readability/else-after-return>` check to ignore

>   `consteval if` condition, where need to retain the else statement.

Reviewer: ```suggestion
  `if consteval` statements, for which the `else` branch must not be removed.
```
